src/Desktop/UIAutomation/A11yAutomation.cs

```diff
@@ -124,6 +143,26 @@ private static DesktopElement ElementFromUIAElement(IUIAutomationElement uia)
             return null;
         }
 
+        /// <summary>
+        /// Get DesktopElements from UIAElements.
+        /// </summary>
+        /// <param name="uia"></param>
+        /// <returns>An IEnumerable of <see cref="DesktopElement"/></returns>
+        private static IEnumerable<DesktopElement> ElementsFromUIAElements(IUIAutomationElementArray elementArray)
+        {
+            for (int i = 0; i < elementArray.Length; ++i)
+            {
+                var uiaElement = elementArray.GetElement(i);
+                if (DesktopElement.IsFromCurrentProcess(uiaElement)) continue;
```

Comment:
Do we need this guard? The guard in `ElementFromUIAElement` exists because that code gets called in response to mouse hover and we explicitly exclude AIWin's process. I'm not sure it applies in the case that we're addressing here

Reply:
We may not need it. But it will now be checked by ElementFromUIAElement and I don't think it will hurt anything.